icu4c/source/common/dictbe.h

```diff
@@ -362,10 +369,19 @@ class CjkBreakEngine : public DictionaryBreakEngine {
      * @internal
      */
   UnicodeSet                fHangulWordSet;
+  UnicodeSet                fHiraganaWordSet;
+  UnicodeSet                fNumberWordSet;
 
   DictionaryMatcher        *fDictionary;
   const Normalizer2        *nfkcNorm2;
 
+ private:
+ /**
+  * <p> Load Japanese particles and auxiliary verbs into fParticleSet.
```

Comment:
Rename to loadJapaneseParticleAndAuxVerbs

Reply:
Resolved